tests/test_samples_rest.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for REST samples."""
+
+import unittest
+
+import dotenv
+from requests import HTTPError, JSONDecodeError
+
+from samples.rest.BingWebSearchV7 import web_search_basic
+
+# from samples.rest.BingAutosuggestV7 import
+
+
+class RESTSamplesTest(unittest.TestCase):
+    """Class for testing REST samples."""
+
+    def setUp(self):
+        self.env = dotenv.dotenv_values()
+
+    def test_web_search_basic(self):
+        """Test the basic REST call to Web Search API"""
+        response = web_search_basic(subscription_key=self.env.get(
+            'BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY'))
```

Comment:
The testcase will fail if the environment variable has different name for any reason. I suggest using (global) variables for entries that you can import and use in all files to access the environment variable.

Reply:
I think I'm skipping this, let me know if you think it is mandatory.

Reply:
It is not mandatory. In my opinion, it is preferred. You can skip it.